HISTORY.md

```diff
@@ -213,6 +215,24 @@
 
 * chore: prepare 15.0 alpha (#4129)
 
+## 14.0.243 beta 2021-02-12
+
+* change(ios/app): Generate offline help from markdown (#4470)
+* fix(windows): tsysinfox64 not signed (#4486)
+* chore(android/app): Update help formatting and images (#4485)
+* fix(android/engine): Display welcome.htm help within the app (#4477)
+* fix(ios): tutorial's link to "Add a Keyboard" links directly to keyboard search (#4491)
+* feat(linux): Improve output of km-package-list-installed (#4481)
+
+## 14.0.242 beta 2021-02-11
+
+* fix(android/engine): Display online keyboard help (#4462)
+* fix(windows): Track modifier changes in UWP apps (#4468)
+* fix(common/resources): Fix help.keyman.com path for commit (#4469)
+* fix(common): create GitHub comments serially (#4472)
+* feat(common/models): mid-context suggestions & reversions, fix(common/models): correction-search SMP issues (#4427)
```

Comment:
#4427 isn't merged yet.

Reply:
I think this relates to https://github.com/keymanapp/keyman/pull/4427#issuecomment-777681199... which I am investigating in #4483...